app/views/admin/calendars/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>New Calendar</h1>
+<%= simple_form_for @calendar, url: admin_calendars_path(@calendar)  do |form| %>
+
+  <% if @calendar.calendar_type %>
+    <p>Regional Calendar</p>
+    <%= form.hidden_field :calendar_type, value: :regional %>
```

Comment:
Can we just use the value that's set here?

Reply:
Rather than defaulting to regional

Reply:
Currently the only link to create a calendar, sets the calendar type to regional, so I've set it up so that if something has already been set, the user doesn't see the options. I don't *think* at the moment, we want a user trying to create a national or school calendar at all.